build.gradle

```diff
@@ -31,9 +31,14 @@ subprojects {
     }
 
     dependencies {
-        errorprone 'com.google.errorprone:error_prone_core:2.3.1'
-        if (JavaVersion.current() == JavaVersion.VERSION_1_8) {
-            errorproneJavac("com.google.errorprone:javac:9+181-r4173-1")
+        errorprone 'com.google.errorprone:error_prone_core:2.18.0'
+    }
+
+    tasks.withType(JavaCompile).configureEach {
+        options.encoding = "utf-8"
+        options.compilerArgs = ["-Xlint:all", "-Xlint:-path", "-Xlint:-options", "-Werror"]
+        options.errorprone {
+            disable("EmptyBlockTag", "InlineMeSuggester")
```

Comment:
I haven't upgraded gradle in neonmarker yet, but I did recently put together a PR which addresses dependency updates. Should I be doing something like this too?

https://github.com/kingthorin/neonmarker/pull/25

Reply:
You should not need to disable any check generally, better have all checks enabled and address the warn/errors. I disabled those because they were causing errors in the generated code.

Reply:
Thanks